pipelines/run-pipelines-on-apache-airflow/README.md

```diff
@@ -0,0 +1,332 @@
+    :type xcom_push: bool
+    :param env: If env is not None, it must be a mapping that defines the
+        environment variables for the new process; these are used instead
+        of inheriting the current process environment, which is the default
+        behavior. (templated)
+    :type env: dict
+    :param output_encoding: Output encoding of bash command
+    :type output_encoding: str
+   ...
+   ```
+
+In Apache Airflow, the output of a component can be used as a property value for any downstream node. (A downstream node is a node that is connected to and executed after the node in question). The pipeline editor renders a selector widget for each property that allows you to choose between two options as a value:
+- A raw value, entered manually
+- The output of an upstream node
+    
+![Property value options for an Airflow node](doc/images/selection-widget.png)
+
+1. The contents of the file requested by the `SimpleHttpOperator` are made available to the downstream nodes in the pipeline by setting the `xcom_push` property of `SimpleHttpOperator` to True. This output value will be the input of the `bash_command` property. Choose '`Please select an output from a parent :`' from the dropdown menu and select `SimpleHttpOperator`. 
+   
+   Since the '`BashOperator`' node is only connected to one upstream node ('`SimpleHttpOperator`'), you can only choose the output of that node. If a node is connected to multiple upstream nodes, you can choose the output of any of these nodes as input, as shown in this example:
+
+   ![Selecting outputs from upstream nodes](doc/images/upstream-nodes-example.png) 
+
+   The output of the `EmailOperator` node cannot be consumed by the '`SlackAPIPostOperator`' node, because the two nodes are not connected in this pipeline. Ensure that the `xcom_push` property is set to `True` for any node whose output will be used in a subsequent node.
+   
+   > Elyra intentionally only supports explicit dependencies between nodes to avoid potential usability issues.
+   
+1. The bash command requested and returned by the `SimpleHttpOperator` node includes an environment variable called `name` that can be set by the `env` property of the `BashOperator`. Enter `{'name': 'World'}` as the value for this field. You can use another name in place of 'World', if desired.
+
+   ![Configure bash node](doc/images/configure-bash-node.png)
+
+1. Save the pipeline.
+
+   ![Save the pipeline](doc/images/save-the-pipeline.png)
+
+1. Rename the pipeline to something meaningful:
+   - right click on the pipeline editor tab and select `Rename Pipeline...` OR
+   - in the JupyterLab File Browser right click on the `.pipeline` file
+
+   ![Rename the pipeline](doc/images/rename-the-pipeline.png)
+
+Next, let's run the pipeline!
+
+### Run the pipeline
+    
+To run the pipeline on Apache Airflow:
+    
+1. Click the `Run` button in the pipeline editor toolbar.
+
+   ![Run the pipeline](doc/images/run-the-pipeline.png)
```

Comment:
This image is showing 'save' instead of 'run'.

Reply:
Looks like I have them switched